### ISSUE TITLE
train_baseline result and provided pre-trained model on baseline++ with ResNet10

### ISSUE THREAD

**Issue by TianyuanYu** [hytseng0509/CrossDomainFewShot#4]
Hi, thanks for sharing your code. When I use the pertrained baseline++ with ResNet10 (downloaded according to Readme), I can always get a very good result combined with gnn (around 63% for 1 shot on MiniImageNet). But when I try training ResNet10 by myself using train_baseline.py, the result combined with gnnnet will drop 4-5% (I tried 3 times with different baseline++ and only to get 58%-59%). Do you have any idea about the possible reason?

**Comment by hytseng0509** [comment]
Hmm interesting.
Can you show the screenshot of the loss curve (baseline++ training) displayed on the tensorboard?
The path of the saved tensorboard file should be indicated in this [line](https://github.com/hytseng0509/CrossDomainFewShot/blob/b03912ce4a8b003240a9cff4c2aee9020ee16db6/train_baseline.py#L61).

**Comment by TianyuanYu** [comment]
![baseline++1](https://user-images.githubusercontent.com/32092577/75673875-db684d00-5c7b-11ea-993a-eeb45c61056b.png)

Thanks for your reply. This is my loss curve for baseline++.

**Comment by hytseng0509** [comment]
Sorry for the late reply.
![image](https://user-images.githubusercontent.com/20281576/76245409-11b44800-61f9-11ea-895b-842002a96068.png)

Seems like your loss is a bit higher than mine. What's the Pytorch version you use?

**Comment by TianyuanYu** [comment]
Thanks. I am using python3.6.7 and pytorch1.4.0
